doc/example/patterns_static_objects.cpp

```diff
@@ -28,13 +29,14 @@ namespace
         return os << "my_class " << c->i_; // the 'c' pointer must be valid when logging
     }
 
-    MOCK_FUNCTION( f, 1, void( my_class* ) ) // being static 'f' outlive the test case
+    MOCK_FUNCTION( f, 1, void( my_class* ) ) // being static 'f' outlives the test case
 }
 
 BOOST_AUTO_TEST_CASE( static_objects_problem )
 {
     my_class c( 42 );
     MOCK_EXPECT( f ).once().with( &c ); // the set expectation will also outlive the test case and leak into other test cases using 'f'
+    ASSERT_VERIFY_FAIL(); // Check that mock::verify fails, but that means other test fail too
```

Comment:
Again this line will show up in the docs but probably make little sense for the user and may confuse her?

Reply:
See above. I wanted to a) in the docs show when and what is expected and b) test this

Reply:
I got really no better idea. Could use a fixture but that can be more confusing as the solution is to use a fixture.